02_bubble/CameronSchafer/bubble.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	// requirement was to sort a copy of the array.
+	t := make([]int, len(s))
+	copy(t, s)
+	check := false
+
+	for !check {
+		tempCheck := true
```

Comment:
just use `check` better name; `swapped` and invert logic.

Reply:
done 👍